cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/queue/AbstractQueueSpecificationGenerator.java

```diff
@@ -72,7 +72,7 @@
       }
 
       if (connection.getSourceType() == FlowletConnection.Type.STREAM) {
-        builder.add(createSpec(QueueName.fromStream(outputName),
+        builder.add(createSpec(QueueName.fromStream(app.getNamespaceId(), outputName),
```

Comment:
It's difficult to use Id.Stream in the places where the implementation of queues is used for streams. We can refactor that out, but I think that this implementation of streams isn't used, its not worth the work.